src/providers/mod.rs

```diff
@@ -130,7 +132,18 @@ fn write_ssh_keys(user: User, ssh_keys: Vec<PublicKey>) -> Result<()> {
                 format!("failed to write to file {:?}", temp_file.path().display())
             })?;
         }
-
+        let name = user.name().to_string_lossy();
+        let path = file_path.to_string_lossy();
+        let mut map: HashMap<&str, &str> = HashMap::new();
+        let messge = format!(
+            "wrote ssh authorized keys file for user:{user}",
+            user = &name
+        );
+        map.insert("AFTERBURN_USER_NAME", &name);
+        map.insert("AFTERBURN_PATH", &path);
+        map.insert("MESSAGE_ID", AFTERBURN_SSH_ATHORIZED_KEYS_MESSAGEID);
+        let res = logging::journal_send(logging::Priority::Info, &messge, map.iter());
+        res.chain_err(|| format!("failed to send journald entry"))?;
```

Comment:
You aren't further using `res`, so you can merge this line with the one above and drop the variable assignment altogether.